ElrondNetwork/elrond-go
# fix meta genesis block

## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
- Fix the problem with the hash of the `metachain` genesis block.
  
## Proposed Changes
- Changed the order when the `validatorStatsRootHash` is set on the `metchain` genesis block.

## Testing procedure
- Fetch from `gateway` or `Elasticsearch` the genesis block ( block with nonce 0 from metachain ) and check if the block's hash is equal with the `previousBlockHash` of the block with the nonce 1.
